lte/gateway/c/oai/common/log.c

```diff
@@ -1478,6 +1481,9 @@ void log_message_int_prefix_id(
   if (!(g_oai_log.is_async)) {
     _LOG_FREE_ITEM(sync_context_p);
   } else {
+    if (async_context_p == NULL) {
+      return;
+    }
```

Comment:
same here, move to after line  1458

Reply:
Same null-pointer access path as above comment, please take a look